docs/modules/ROOT/pages/how-tos/configuring/prefetching-dependencies.adoc

```diff
@@ -204,6 +204,60 @@ spec:
 * From the {ProductName} *Applications* view, go to *Activity > Latest commits*.
 //include::../partials/con_hermetic_verification.adoc[]
 
+=== Prefetching `pip` dependencies from custom index servers
+
+Cachi2 supports the link:https://pip.pypa.io/en/stable/cli/pip_install/#install-index-url[--index-url] option.
```

Comment:
Is this correct? `--index-url`

or do we need to use:

```suggestion
Cachi2 supports the link:https://pip.pypa.io/en/stable/cli/pip_install/#install-index-url[index url] option.
```

Reply:
`--index-url` is how the pip documentation refers to it and how it appears in the requirements.txt file